libcudacxx/include/cuda/__functional/maximum.h

```diff
@@ -23,6 +23,8 @@
 
 #include <cuda/std/__type_traits/common_type.h>
 
+#include <cuda/std/__cccl/prologue.h>
```

Comment:
Idea: This would definitely be for a follow up PR, but we should consider adding a test that scans libcu++ headers and verifies that these includes exist so we don't forget them in new headers. We have some machinery that does similar checks for Thrust/CUB: https://github.com/NVIDIA/cccl/blob/main/thrust/testing/cmake/check_source_files.cmake